modules/sublimeBidAdapter.js

```diff
@@ -187,30 +180,38 @@ function interpretResponse(serverResponse, bidRequest) {
       pbav: SUBLIME_VERSION
     };
 
-    sendEvent('bida', true);
     bidResponses.push(bidResponse);
-  } else {
-    sendEvent('dnobid');
   }
 
   return bidResponses;
 }
 
+/**
+ * Send pixel when bidWon event is triggered
+ * @param {Object} timeoutData
+ */
+function onBidWon(bid) {
+  log('Bid won', bid);
+  sendEvent('bidwon', true);
```

Comment:
Ne correspond pas à la signature de sendEvent.

```suggestion
  sendEvent('bidwon');
```